src/piggui.rs

```diff
@@ -142,6 +144,10 @@ pub enum Message {
     None,
     HardwareListener(HWListenerEvent),
     ChangeOutputLevel(LevelChange),
+    Add,
```

Comment:
Maybe add some noun to the Messages?
like ToastClose, ToastTimeout etc?

I see some projects when they get bigger, nest Messages
so they would have 

enum ToastMessage {
   Close, 
   Timeout,
   ...etc 
}

enum Message {
   Toast(ToastMessage),
   ....
}

Then code can be 
     ToastMessage(tm) => Toast::update(tm)

and you can move all the handling of the toast messages into the Toast struct, in a different file.

Otherwise, for large Uis, the amount of Messages get's huge, and the update method get's huge....